.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "adapter/python"]
+	path = adapter/python
+	url = https://github.com/MatthiasPreuner/ecco-adapter-python.git
```

Comment:
a submodule for the isse ecco project should have its repository also inside the isse org

Reply:
should we make an adapter repository on isse org?